scrapers_next/tn/committees.py

```diff
@@ -0,0 +1,41 @@
+from spatula import HtmlPage, HtmlListPage, CSS, XPath, SelectorError, SkipItem, URL, ListPage
+from openstates.models import ScrapeCommittee
+
+
+class SenateCommitteeDetail(HtmlPage):
+    example_source = "https://wapp.capitol.tn.gov/apps/CommitteeInfo/SenateComm.aspx?ga=113&committeeKey=620000"
+
+    def process_page(self):
+        com = self.input
+        com.add_source(self.source.url)
+        com.add_link(self.source.url, note="homepage")
+        
+        class_officers = self.root.xpath(".//ul[@class='no-list members large small-block-grid-3']//a")
+        for officer in class_officers:
+            name_and_title = [x.strip() for x in officer.text_content().split("\r\n") if len(x)]
+            name, title = name_and_title
+            com.add_member(name, title)
+    
+        try:
```

Comment:
This is quite the `try` block. Any chance we could reduce the number of commands in this block?

Reply:
I made changes to the try and except block to adjust for your comments. Please let me know if there is anything else you'd like me to change.